react-app/src/components/ResultSidebar.js

```diff
@@ -15,6 +15,37 @@ function ResultSidebar(props) {
     // Save the results permanently somehow
   }
 
+  function getLoggerStats() {
+    let errors = 0;
+    let warnings = 0;
+    let debugs = 0;
+    let total = 0;
+
+    props.loggedMessages.forEach((log) => {
+      if (log.level === 'error') {
+        errors += 1;
+      }
+      if (log.level === 'warn') {
+        warnings += 1;
+      }
+      if (log.level === 'debug') {
+        debugs += 1;
+      }
+      total += 1;
+    });
+
+    return (
+      <Accordion.Body>
+        <p key={0}>Errors: {errors}</p>
+        <p key={1}>Warnings: {warnings}</p>
+        <p key={2}>Debug Messages: {debugs}</p>
+        <p className="emphasized-list-text" key={3}>
+          Total Messages: {total}
```

Comment:
Since the total count of resources was moved to the top, we should move the total messages count to match.

Reply:
I've moved the total message count. It uses the same CSS class as the total resource count text, so the color will update during rebasing.